src/cli_plugin/install/__tests__/kibana.js

```diff
@@ -0,0 +1,108 @@
+import expect from 'expect.js';
+import sinon from 'sinon';
+import Logger from '../../lib/logger';
+import { join } from 'path';
+import rimraf from 'rimraf';
+import mkdirp from 'mkdirp';
+import { existingInstall, checkVersion } from '../kibana';
+
+describe('kibana cli', function () {
+
+  describe('plugin installer', function () {
+
+    describe('kibana', function () {
+      const testWorkingPath = join(__dirname, '.test.data');
```

Comment:
My previous comments about in-memory file system mocks or tmp directories apply here as well.